test_calculus.py

```diff
@@ -1,9 +1,15 @@
 """
 test_calculus.py
 """
+
+import numpy as np
 import calculus as calc
+from calculus import scipy_simpson
```

Comment:
DO not make a second import statement of the same module, which is not even used